### PR TITLE
std.string.capitalize_unittest_example

### PR DESCRIPTION
**Pull request by burner**
* added unittest doc example

### CODE REVIEW

File: std/string.d [hunk 1]
```diff
@@ -2224,6 +2224,13 @@ S capitalize(S)(S s) @trusted pure
     return changed ? cast(S)retval : s;
 }
 
+///
+unittest
```
Comment (JackStouffer) [inline std/string.d]: Nitpick: why not annotate with `@trusted pure` like the other unit test?

Comment (burner) [inline std/string.d]: I'll add pure, even though it does not show up in the output. trusted has no meaning

**Comment by JakobOvrum** [comment]
LGTM

**Comment by andralex** [comment]
thx!

**Comment by andralex** [comment]
Auto-merge toggled on

**Comment by burner** [comment]
thanks

**Pull request merged**